docs/development/coding-standard.rst

```diff
@@ -1,5 +1,5 @@
-Twisted Coding Standard
-=======================
+Coding Standard
```

Comment:
This was just a drive-by comment...as in the table of content each item was starting with "Twisted ..." and it was kind of redundeand